### DISCUSSION TITLE
Why hasn't the 1.6 milestone been released yet?

### DISCUSSION THREAD

**Discussion by Anufant** [graph-gophers/graphql-go#631] [General]
Hello!
Sorry if my question is stupid.
I see all issues for milestone v1.6 were closed long time ago, but 1.6 has not yet been released?

**Comment by pavelnikolov** [comment]
I'm planning to release it in the next week or two.

**Reply by ralfkrakowski** [reply]
Any progress on this issue?

**Reply by pavelnikolov** [reply]
The reason I still haven't released `v1.6` is because I am still not happy with the directives implementation. What I'm considering right now is releasing the current code without the directives and then work on the directives implementation for `v1.7`.

**Reply by ralfkrakowski** [reply]
Ok, thanks for letting me know. We use graphql-go and we are very happy with it.  Don't let me push you to do a release, do it when you feel it's alright.

**Reply by pavelnikolov** [reply]
Thank you for the nice words and your patience. I do realize that it's been quite a while and this release is way overdue.
